zerver/lib/actions.py

```diff
@@ -5101,9 +5101,11 @@ def send_message_moved_breadcrumbs(
     user_profile: UserProfile,
     old_stream: Stream,
     old_topic: str,
+    old_thread_notification_string: str,
+    send_notification_to_old_thread: bool,
     new_stream: Stream,
     new_topic: Optional[str],
-    send_notification_to_old_thread: bool,
+    new_thread_notification_string: str,
```

Comment:
I think a better interface would be to pass an `Optional[str]`, and treat `None` as the boolean value being false, rather than passing two semi-duplicate fields.

Reply:
Makes sense! Updated.